src/main/SearchForm.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { useState } from "react/cjs/react.development";
+import "./SearchForm.css";
+
+const beersUrl = `https://api.punkapi.com/v2/beers`;
```

Comment:
Once you put the same constant from `Catalog.js` into some generic file with constants you will no longer need this one.